cookbook/case_studies/in_container.mk

```diff
@@ -11,10 +11,6 @@ $(SERIALIZED_PB_OUTPUT_DIR): clean
 serialize: $(SERIALIZED_PB_OUTPUT_DIR)
 	pyflyte --config /root/sandbox.config serialize workflows -f $(SERIALIZED_PB_OUTPUT_DIR)
 
-.PHONY: register
```

Comment:
Should we remove fast_register too?

Reply:
no we can wait for flytectl fast register